src/core_plugins/kibana/public/dashboard/styles/index.less

```diff
@@ -112,6 +112,10 @@ dashboard-grid {
   i.remove {
     cursor: pointer;
   }
+
+  .gs-w {
+    z-index: auto;
```

Comment:
Is something else setting this to something besides `auto`? `auto`  should be the default `z-index` for elements.

Reply:
Gridster sets this to 1 for each grid. This caused problems with overflow of the list select and needed to be set to auto